pstress/pstress-run.sh

```diff
@@ -92,6 +97,23 @@ echoit(){
   if [ ${WORKDIRACTIVE} -eq 1 ]; then echo "[$(date +'%T')] [$SAVED] $1" >> /${WORKDIR}/pquery-run.log; fi
 }
 
+# Kill the server
+kill_server(){
+  SIG=$1
+  echoit "Killing the server with Signal $SIG";
+  { kill -$SIG ${MPID} && wait ${MPID}; } 2>/dev/null
+}
+
+# PXC Bug found display function
+pxc_bug_found(){
+  if [ "$(${SCRIPT_PWD}/search_string.sh ${RUNDIR}/${TRIAL}/node1/node1.err 2>/dev/null)" != "" ]; then
```

Comment:
this is wrong. what would be the benefit of moving code like this? It does not decrease the duplication  of code
Instead, it should be a method that takes node number and argument and searches for error.

Reply:
Ok. I have fixed it.